data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/strategy/UnorderedConsumerRecordOffsetStrategy.java

```diff
@@ -19,21 +19,28 @@
 
 import dev.knative.eventing.kafka.broker.dispatcher.ConsumerRecordOffsetStrategy;
 import io.micrometer.core.instrument.Counter;
-import io.vertx.core.Future;
 import io.vertx.kafka.client.common.TopicPartition;
 import io.vertx.kafka.client.consumer.KafkaConsumer;
 import io.vertx.kafka.client.consumer.KafkaConsumerRecord;
 import io.vertx.kafka.client.consumer.OffsetAndMetadata;
+import java.util.HashMap;
 import java.util.Map;
 import java.util.Objects;
+import java.util.SortedSet;
+import java.util.TreeSet;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+/**
+ * This class implements the offset strategy that makes sure that, even unordered, the offset commit is ordered.
+ */
 public final class UnorderedConsumerRecordOffsetStrategy implements ConsumerRecordOffsetStrategy {
 
   private static final Logger logger = LoggerFactory
     .getLogger(UnorderedConsumerRecordOffsetStrategy.class);
 
+  private final Map<TopicPartition, Long> lastAckedMap;
+  private final Map<TopicPartition, SortedSet<Long>> pendingAcksMap;
```

Comment:
A little comment of what these are would help.

Reply:
Ok I've added some comments and renamed the maps to be easier to understand